lib/importer/registry/bootstrap/0.0.0.json

```diff
@@ -2,8 +2,16 @@
   "dependencies": [{
     "name": "bootstrap",
     "path": "../node_modules/bootstrap/dist",
-    "main": "js/bootstrap.min",
+    "main": "js/bootstrap.min.js",
     "deps": ["jquery"],
-    "exports": "$"
-  }]
+    "exports": "$",
+    "resources": [
+      "css/bootstrap.css"
+    ]
+  }],
+  "tutorial": [
+    "1. in main.js add \"import 'bootstrap';\"",
+    "2. in app.html add \"<require from='bootstrap/css/bootstrap.css'></require>\"",
+    "3. Make sure that jquery is installed (or do 'au install jquery') and make sure that the dependency in aurelia.json is above the bootstrap dependency, order matters!"
```

Comment:
"3. Make sure that jquery is installed (or do 'au install jquery') and make sure that the dependency in aurelia.json is above the bootstrap dependency, order matters!"

---

"3. Make sure that jQuery is installed ('au install jquery') and the dependency definition in aurelia.json is preceding the bootstrap dependency since order matters!"